examples/specs/normalized/layer_likert_normalized.vl.json

```diff
@@ -0,0 +1,323 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v3.json",
+  "description": "Likert Scale Ratings Distributions and Medians. (Figure 9. from @jhoffswell and @zcliu's 'Interactive Repair of Tables Extracted from PDF Documents on Mobile Devices' -- http://idl.cs.washington.edu/files/2019-InteractiveTableRepair-CHI.pdf)",
```

Comment:
```suggestion
  "description": "Likert Scale Ratings Distributions and Medians. (Figure 9. from @jhoffswell and @zcliu's ['Interactive Repair of Tables Extracted from PDF Documents on Mobile Devices'](https://idl.cs.washington.edu/files/2019-InteractiveTableRepair-CHI.pdf))",
```

Reply:
Nah, this is description in the JSON -- why using markdown?

Reply:
Because we can pull it out into descrtions in other places and it's still human readable. Otherwise, just remove the title and have only the link.